apps/web/src/state/pools/helpers.ts

```diff
@@ -42,29 +42,29 @@ const transformProfileRequirement = (profileRequirement?: { required: boolean; t
     : undefined
 }
 
-export const transformPool = (pool: SerializedPool): Pool.DeserializedPool<Token> => {
+export const transformPool = (pool: SerializedPool): DeserializedPool<Token> => {
   const {
     totalStaked,
     stakingLimit,
-    numberBlocksForUserLimit,
+    numberSecondsForUserLimit,
     userData,
     stakingToken,
     earningToken,
     profileRequirement,
-    startBlock,
+    startTimestamp,
     ...rest
   } = pool
 
   return {
     ...rest,
-    startBlock,
+    startTimestamp,
```

Comment:
hmm. For BSC, it should be block-based caluclation instead of timestamp, right?

Reply:
Yes and it's already converted from block to timestamp on bsc

Reply:
Can refer https://github.com/pancakeswap/pancake-frontend/pull/6600/files#diff-5d7737987d8acc78406f7c7c03dafcaf248abeba85b13dfd87d08a169834cc09R124